components/Notifications.tsx

```diff
@@ -73,11 +73,10 @@ export default function Notifications() {
                             Copy tx hash
                           </PillAction>
                           <a
-                            className="ml-2"
-                            href={`https://etherscan.io/tx/${hash}`}
+                            href={getBlockExplorerUrl(hash as string, chainId as number)}
```

Comment:
suggestion: i'd remove the typecast on `hash` and if possible on `chainId`. if we have to cast them it may be a code smell. happy to chat about this if you'd like!